### PR TITLE
Code Review Updates - Added Test

### PR DESCRIPTION
**Pull request by sethwhite3**
These are all my changes!

Analysis: 
Was the program available in UC Github on time? Yes!
Is the program documented/commented well enough for you to understand? Yes in the readme.
Does the program compile? Yes

This group added me very quickly to their github. Compiled and worked great. 

Concepts I learned:
ListAdapters. I've implemented this in our project and I feel we were having some issues or afraid I never correctly did it right so its good to see another group using it and learning how it was implemented.

Secondly, separating out their fragments. We don't have a lot in our project at the moment. But currently my ListAdapter is in the same place as my MainActivity. Not a big deal I'm sure, but good to see they are organizing project well so far. 

My changes consist of including Test. To start them on the path of including more test in their project. Their progress so far didn't have many areas to fix or include new things without developing new features which isn't the idea of a review. So, I did what I could at the moment.

Overall, it was a great start for their project. I look forward to seeing it progress. Great work!!

### CODE REVIEW

File: app/src/test/java/edu/uc/hickmadc/customnotifications/ExampleUnitTest.kt [hunk 1]
```diff
@@ -14,4 +18,23 @@ class ExampleUnitTest {
...
+        var groupTest: Group = Group("Testing Group", "Test for class", true, false, 1)
+        assertEquals("Testing Group", groupTest.toString());
+    }
+
+    @Test
+    fun confirmsSchedule_outputsSchedule () {
+        val cal = Calendar.getInstance()
+        var scheduleTest: Schedule = Schedule(cal, 5, 1)
+        assertEquals(cal, scheduleTest.original_occurence);
+    }
```
Comment (sethwhite3) [inline app/src/test/java/edu/uc/hickmadc/customnotifications/ExampleUnitTest.kt:39]: Testing your classes and the start of creating some Test for your project!

File: app/src/test/java/edu/uc/hickmadc/customnotifications/ExampleUnitTest.kt [hunk 2]
```diff
@@ -14,4 +18,23 @@ class ExampleUnitTest {
     fun addition_isCorrect() {
         assertEquals(4, 2 + 2)
     }
```
Comment (discospiff) [inline app/src/test/java/edu/uc/hickmadc/customnotifications/ExampleUnitTest.kt:20]: Remove this.

File: app/src/test/java/edu/uc/hickmadc/customnotifications/ExampleUnitTest.kt [hunk 3]
```diff
@@ -1,8 +1,12 @@
 package edu.uc.hickmadc.customnotifications
```
Comment (discospiff) [inline app/src/test/java/edu/uc/hickmadc/customnotifications/ExampleUnitTest.kt:1]: Rename the file, as it is actually performing tests now.  "Example" is no longer appropriate.

**Comment by discospiff** [comment]
Unit test coverage is always good.  This concept is worth merging and expanding.

**Pull request merged**